javascript/javascript1/week1/family-pizza.js

```diff
@@ -0,0 +1,14 @@
+
+const pizzasPrice = 70;
+let amountOfPizzas = 3;
+let pizzaSize = 'Family Pizza';
+
+if (pizzaSize)
+{
+    totalPrice = 2 * pizzasPrice;//Family pizza price is double the price of normal pizza
+}
+
+console.log("New pizza order:"+" " +amountOfPizzas+" "+ pizzaSize+"."+"\n"+"Total cost of Order is:"+" "+ (3 * totalPrice));
```

Comment:
Good to see that you use the newline here. 
Consider using backticks for interpolation.